src/content/en/ilt/pwa/working-with-indexeddb.md

```diff
@@ -31,7 +31,7 @@ This text guides you through the basics of the  [IndexedDB API](https://develope
 
 
 
-IndexedDB is a large-scale, noSQL storage system. It lets you store just about anything in the user's browser. In addition to the usual search, get, and put actions, IndexedDB also supports transactions. Here is the definition of IndexedDB on MDN:
+IndexedDB is a large-scale, NoSQL storage system. It lets you store just about anything in the user's browser. In addition to the usual search, get, and put actions, IndexedDB also supports transactions. Here is the definition of IndexedDB on MDN:
```

Comment:
This file is ingested from a GoogleDoc source, so changes will be lost on our next update.

@DavidScales, @nasearle, @agektmr: fyi to update the original doc